src/main/webapp/app/tasks/local-task.service.ts

```diff
@@ -1,30 +1,41 @@
-import { Injectable } from '@angular/core';
-import { Observable, of } from 'rxjs';
-import { v4 as uuid } from 'uuid';
+import { Injectable } from "@angular/core";
+import { Observable, of } from "rxjs";
+import { v4 as uuid } from "uuid";
 
-import { Task } from './task';
-import { TaskService } from './task.service';
+import { Task } from "./task";
+import { TaskService } from "./task.service";
+import { sortTasksByStatus } from "./utils";
 
 @Injectable()
 export class LocalTaskService implements TaskService {
-
-  private static readonly STORAGE_KEY: string = 'tiny.tasks';
+  private static readonly STORAGE_KEY: string = "tiny.tasks";
 
   getAll(): Observable<Task[]> {
     return of(this.readTasks());
   }
 
   create(name: string): Observable<Task> {
     const tasks = this.readTasks();
-    const task = {id: uuid(), name};
+    const task = { id: uuid(), name };
     tasks.push(task);
     this.writeTasks(tasks);
     return of(task);
   }
 
+  update(task: Task): Observable<Task> {
+    const tasks = this.readTasks();
+    const index = tasks.findIndex((t) => t.id == task.id);
```

Comment:
probably should use === just to be on the safe side

Reply:
Definitely. This is a typo that I made accidentally..

Reply:
Definitely! I missed that.